src/overlays/ovl_i2/fox_me.c

```diff
@@ -1,97 +1,2527 @@
+                player->camAt.z = player->unk_138;
+
+                for (i = 0; i < 4; i++) {
+                    Object_Kill(&gActors[i].obj, &gActors[i].sfxPos);
+                }
+
+                D_80178488 = 1;
+                break;
+            }
+    }
+
+    switch (D_80177A80) {
+        case 500:
+            if ((gTeamShields[2] > 0) && (gTeamShields[3] > 0)) {
+                func_800BA808(gMsg_ID_3005, RCID_SLIPPY);
+            }
+            break;
+
+        case 600:
+            if ((gTeamShields[2] > 0) && (gTeamShields[3] > 0)) {
+                func_800BA808(gMsg_ID_3010, RCID_PEPPY);
+            }
+            break;
+
+        case 700:
+            func_800BA808(gMsg_ID_3015, RCID_FOX);
+            break;
+    }
+
+    Math_SmoothStepToF(&player->camEye.x, D_80177978, D_80177A48[0], 20000.0f, 0.0f);
+    Math_SmoothStepToF(&player->camEye.y, D_80177980, D_80177A48[0], 20000.0f, 0.0f);
+    Math_SmoothStepToF(&player->camEye.z, D_80177988, D_80177A48[0], 20000.0f, 0.0f);
+    Math_SmoothStepToF(&player->camAt.x, D_801779A0, D_80177A48[0], 20000.0f, 0.0f);
+    Math_SmoothStepToF(&player->camAt.y, D_801779B8, D_80177A48[0], 20000.0f, 0.0f);
+    Math_SmoothStepToF(&player->camAt.z, D_801779C0, D_80177A48[0], 20000.0f, 0.0f);
+}
+
+void func_i2_8018D9EC(Actor* actor) {
+    switch (actor->unk_0B8) {
+        case 0:
+            actor->fwork[0] = (Rand_ZeroOne() - 0.5f) * 10.0f;
+            actor->unk_0B8 = 1;
+
+        case 1:
+            actor->obj.rot.z += actor->fwork[0];
+            break;
+    }
+    func_i2_80187530(actor);
+}
+
+void func_i2_8018DA6C(Actor* actor) {
+    RCP_SetupDL_60(gFogRed, gFogGreen, gFogBlue, gFogAlpha, gFogNear, gFogFar);
+    gSPDisplayList(gMasterDisp++, D_6017AD0);
+}
+
+void func_i2_8018DAEC(Actor* actor) {
+    RCP_SetupDL_60(gFogRed, gFogGreen, gFogBlue, gFogAlpha, gFogNear, gFogFar);
+    gSPDisplayList(gMasterDisp++, D_6016240);
+}
+
+void func_i2_8018DB6C(Actor* actor) {
+    RCP_SetupDL_60(gFogRed, gFogGreen, gFogBlue, gFogAlpha, gFogNear, gFogFar);
+    gSPDisplayList(gMasterDisp++, D_600CD60);
+}
+
+void func_i2_8018DBEC(Effect* effect) {
+    if (effect->unk_4E == 1) {
+        Matrix_RotateX(gGfxMatrix, effect->obj.rot.x * M_DTOR, 1);
+        Matrix_Scale(gGfxMatrix, effect->scale2, effect->scale2, effect->scale2, 1);
+        Matrix_RotateZ(gGfxMatrix, (M_PI / 2), 1);
+        Matrix_SetGfxMtx(&gMasterDisp);
+        gDPSetPrimColor(gMasterDisp++, 0, 0, 255, 255, 255, effect->unk_44);
+        gSPDisplayList(gMasterDisp++, D_60263F0);
+    }
+}
+
+void func_i2_8018DCE4(Effect* effect) {
+    if (effect->unk_4E != 0) {
+        Matrix_RotateX(gGfxMatrix, effect->obj.rot.x * M_DTOR, 1);
+        Matrix_Scale(gGfxMatrix, effect->scale2, effect->scale2, effect->scale2 * 3.0f, 1);
+
+        if (gGameFrameCount & 1) {
+            Matrix_RotateZ(gGfxMatrix, M_PI, 1);
+        }
+
+        Matrix_RotateX(gGfxMatrix, -1 * (M_PI / 2), 1);
```

Comment:
```suggestion
        Matrix_RotateX(gGfxMatrix, -M_PI / 2, 1);
```
I don't think these parentheses are needed